starknet_py/net/models/abi/definition_test.py

```diff
@@ -0,0 +1,334 @@
+import json
+from collections import OrderedDict
+
+import pytest
+
+from starknet_py.cairo.type_parser import UnknownCairoTypeError
+from starknet_py.net.models.abi.definition import AbiDefinition, AbiDecodingError
+from starknet_py.cairo.data_types import StructType, FeltType, ArrayType
+from starknet_py.tests.e2e.fixtures.misc import read_contract
+
```

Comment:
It's quite a big file 😓 Let's move the setup (creating objects) to the fixtures directory